### ISSUE TITLE
 Protocol error

### ISSUE THREAD

**Issue by abr4xas** [r0oth3x49/udemy-dl#42]
Hello, Im having this problem can you help me?

```python
Traceback (most recent call last):
  File "udemy-dl.py", line 898, in <module>
    main()
  File "udemy-dl.py", line 688, in main
    udemy.ExtractAndDownload()
  File "udemy-dl.py", line 396, in ExtractAndDownload
    out = self.Downloader(_url, lecture_name, chapter_path)
  File "udemy-dl.py", line 88, in Downloader
    out = course_dl.download(url, title, filepath=path, quiet=True, callback=self.Download)
  File "/$USER/udemy-dl/udemy/_downloader.py", line 90, in download
    outfh = open(temp_filepath, fmode)
IOError: [Errno 71] Protocol error: '/$USER/udemy-dl/vuejs-from-beginner-to-professional/02 Fundamentals of Vue.js/010 Exercises: String interpolation, methods and directives.mp4.part'
```
I'm using this:

```bash
python udemy-dl.py https://www.udemy.com/vuejs-from-beginner-to-professional/
```

**Comment by r0oth3x49** [comment]
@abr4xas thanks for the report. will have a look & fix ASAP.

**Comment by abr4xas** [comment]
@r0oth3x49 nop, i'm using Ubuntu 16.04.3 LTS. With any course.

**Comment by r0oth3x49** [comment]
@abr4xas thanks for explanation & using this script i will test this script on `Ubuntu 16.04.3 LTS` against some free courses will add a fix ASAP.

**Comment by r0oth3x49** [comment]
@abr4xas i have downloaded `Ubuntu 16.04.3 LTS 64-bit` and tested against a free course in my account it worked for me i cannot reproduce the error. have a look below.
![picture](https://user-images.githubusercontent.com/11024397/34041139-b4fe5370-e1b8-11e7-85fe-05818bbd3c29.png)

**Comment by abr4xas** [comment]
It is strange, I had tried with 3 different courses and always the same error...

**Comment by r0oth3x49** [comment]
@abr4xas it took two days to download the latest available `ubuntu-16.04.3-desktop-amd64`. did you installed all the requirements? try installing `requests[security]` or try to download using `virtualenv`.
or try with `python3` may be it works for you.  and one more solution try to give all permissions to all files and folders in `udemy-dl`

**Comment by r0oth3x49** [comment]
@abr4xas one of my friend face the same problem. but according to him that was a human error so i 'm closing this error and will w8 for more similar reports.
